estate/models/house.py

```diff
@@ -0,0 +1,39 @@
+from odoo import models, fields
+
+class house(models.Model):
+    _name = 'house'
```

Comment:
```suggestion
    _name = 'estate.house'
```

for clarity we use a two word model name